app/src/main/kotlin/com/wire/android/ui/common/imagepreview/BulletHoleImagePreview.kt

```diff
@@ -44,6 +48,19 @@ import androidx.constraintlayout.compose.ConstraintLayout
 import coil.compose.rememberAsyncImagePainter
 import com.wire.android.ui.common.dimensions
 import com.wire.android.util.toBitmap
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+@Composable
+private fun loadBiMap(imageUri: Uri): State<Bitmap?> {
```

Comment:
can we set this as a highorder funcntion annotated with stable

```kotlin
@Stable
suspend fun Context.loadBitMap(imageUri: Uri): State<Bitmap?> {
       return produceState<Bitmap?>(initialValue = null, imageUri) {
       withContext(Dispatchers.IO) {
            value = imageUri.toBitmap(this)
        }
}
```

Reply:
It will be hard to pass the returned value from this suspend function to the composable